src/index.js

```diff
@@ -0,0 +1,336 @@
+			} else {
+				simple_local_avatar_frame.setState('cropper');
+			}
+		});
+
+		simple_local_avatar_frame.open();
+	});
+
+	/**
+	 * If the Local Avatar is removed and set to the  default one
+	 */
+	$('#simple-local-avatar-remove').on('click', function (event) {
+		event.preventDefault();
+
+		if (avatar_working) return;
+
+		avatar_lock('lock');
+		$.get(ajaxurl, {
+			action: 'remove_simple_local_avatar',
+			user_id: i10n_SimpleLocalAvatars.user_id,
+			_wpnonce: i10n_SimpleLocalAvatars.deleteNonce,
+		})
+			.done(function (data) {
+				if (data != '') {
+					avatar_container.innerHTML = data;
+					$('#simple-local-avatar-remove').hide();
+					avatar_ratings.disabled = true;
+				}
+			})
+			.always(function () {
+				avatar_lock('unlock');
+			});
+	});
+
+	/**
+	 * Update the Local Avatar image
+	 */
+
+	avatar_input.on('change', function (event) {
+		avatar_preview.attr('srcset', '');
+		avatar_preview.attr('height', 'auto');
+		URL.revokeObjectURL(avatar_blob);
+		if (event.target.files.length > 0) {
+			avatar_blob = URL.createObjectURL(event.target.files[0]);
+			avatar_preview.attr('src', avatar_blob);
+		} else {
+			avatar_preview.attr('src', current_avatar);
+		}
+	});
+});
+
+function avatar_lock(lock_or_unlock) {
+	if (undefined == avatar_spinner) {
+		avatar_ratings = document.getElementById('simple-local-avatar-ratings');
+		avatar_spinner = jQuery(document.getElementById('simple-local-avatar-spinner'));
+		avatar_container = document.getElementById('simple-local-avatar-photo');
+		avatar_form_button = jQuery(avatar_ratings).closest('form').find('input[type=submit]');
+	}
+
+	if (lock_or_unlock == 'unlock') {
+		avatar_working = false;
+		avatar_form_button.removeAttr('disabled');
+		avatar_spinner.hide();
+	} else {
+		avatar_working = true;
+		avatar_form_button.attr('disabled', 'disabled');
+		avatar_spinner.show();
+	}
+}
+
+/**
+ * Returns a set of options, computed from the attached image data and
+ * control-specific data, to be fed to the imgAreaSelect plugin in
+ * wp.media.view.Cropper
+ *
+ * @param {wp.media.model.Attachment} attachment
+ * @param {wp.media.controller.Cropper} controller
+ * @returns {object} Options
+ */
+function simple_local_avatar_calculate_image_select_options(attachment, controller) {
+	const control = controller.get('control');
+
+	const realWidth = attachment.get('width');
+	const realHeight = attachment.get('height');
+
+	let xInit = parseInt(control.params.width, 10);
+	let yInit = parseInt(control.params.height, 10);
+
+	const ratio = xInit / yInit;
+
+	controller.set(
+		'canSkipCrop',
+		!control.mustBeCropped(false, false, xInit, yInit, realWidth, realHeight),
+	);
+
+	const xImg = xInit;
+	const yImg = yInit;
+
+	if (realWidth / realHeight > ratio) {
+		yInit = realHeight;
+		xInit = yInit * ratio;
+	} else {
+		xInit = realWidth;
+		yInit = xInit / ratio;
+	}
+
+	const x1 = (realWidth - xInit) / 2;
+	const y1 = (realHeight - yInit) / 2;
+
+	const imgSelectOptions = {
```

Comment:
We can simply return from here by replacing the line with `return {` and removing the `return imgSelectOptions;` statement at the bottom.